.github/workflows/tests.yml

```diff
@@ -1,19 +1,24 @@
 name: Tests
-
-on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
+on: [push, pull_request]
 
 jobs:
-  all:
-
+  tests:
     runs-on: ubuntu-latest
```

Comment:
Is there a lighter image you can run this on? Or does it not make a difference

Reply:
Shouldn't make a huge diff since it's already pretty fast. If anything this is faster because it's used a lot so might be cached.

Reply:
makes sense